lib/index.js

```diff
@@ -192,6 +202,37 @@ KnexMigrator.prototype.migrateTo = function migrateTo(options) {
     });
 };
 
+/**
+ * will delete the target database
+ *
+ * @TODO:
+ * - think about deleting only the tables
```

Comment:
There are two things that I think need some consideration here:
1. deleting and recreating the whole database (and connection) between every test will result in slower tests for not much value
2. having "reset" delete the database steps a little outside of what the migrator tool possibly ought to have control over. What I mean is, the init & upgrade migrations only ever affect the database in a way set out in the corresponding files. This `reset` command will delete EVERYTHING regardless of whether it was ever created by knex-migrator. It's an edgecase, but people do and will install Ghost and then add their own stuff manually to the database. I think deleting everything in this case would be unexpected?